### DISCUSSION TITLE
Small dataset for Vocoder

### DISCUSSION THREAD

**Discussion by hamidkm9** [coqui-ai/TTS#1713] [General Q&A] [answered]
What is your suggestion for training a Vocoder on a small dataset? Fine-tuning on Multi-band MelGAN does not yield a desirable audio quality. (The TTS has been trained on the small dataset very well.)

**Comment by erogol** [comment, marked as answer]
finetune Univenet or HifiGAN

**Reply by hamidkm9** [reply]
Thank you!
